src/mero_conn.erl

```diff
@@ -69,6 +73,18 @@ delete(Name, Key, Timeout) ->
     pool_execute(PoolName, delete, [Key, TimeLimit], TimeLimit).
 
 
+mdelete(Name, Keys, Timeout) ->
+    TimeLimit = mero_conf:add_now(Timeout),
+    KeysGroupedByShards = mero_cluster:group_by_shards(Name, Keys),
+    rpc:pmap({mero_conn, mdelete_exec}, [Name, TimeLimit], KeysGroupedByShards),
+    ok.
```

Comment:
very nice the rpc_pmap but note that it is not respecting the spec since it always return ok instead of going through the list of results to check if there was any issue or not.